### DISCUSSION TITLE
Apply highlight to whole line

### DISCUSSION THREAD

**Discussion by caenrique** [nvim-neo-tree/neo-tree.nvim#1042] [Ideas]
I'm trying to modify my setup to make better use of colors for indicators. I've started with git diffs, but I also want to extend that to the git status indicators in the file tree. The current git status indicators in neo-tree are too subtle for my liking, so I would like to have a colored background.

I've tried defining different highlights, but those are only applied to the specific nodes (filename, git_status, etc), not the whole width of the line. This would also be great for indicating opened files.

Looking at the code, seems that there isn't the concept of a node that represents the line, so I'm not sure where or how would be a good way to incorporate this changes?

**Comment by cseickel** [comment]
You're right in that there is no concept in neo-tree that fits this need. The only way I can think of to accomplish this is to handle the `after_render` event and manually apply highlights to rows that have the git status you are interested in highlighting.

```
:h neo-tree-events
:h nvim_buf_add_highlight()
```
